<launch>
<arg name="config_file_location" default="$(find zed_cpu_ros)/config/SN1267.conf"/>
<arg name="camera_namespace" default="camera"/>

<node pkg="zed_cpu_ros" type="zed_cpu_ros" name="zed_cpu_ros_node" output="screen" ns="$(arg camera_namespace)" required="true">
	<param name="resolution" value="2"/>
	<param name="frame_rate" value="30"/>
	<param name="config_file_location" value="$(arg config_file_location)"/>
	<param name="show_image" value="false"/>
	<param name="left_frame_id" value="left_frame"/>
	<param name="right_frame_id" value="left_frame"/>
	<param name="load_zed_config" value="true"/>
</node>

<node pkg="tf" type="static_transform_publisher" name="static_tf_1" args="0.25 0 0.4 0 0 0 1 base_link left_frame 30"/>
<!-- <node pkg="tf" type="static_transform_publisher" name="static_tf_2" args="0 0.120 0 0 0 0 1 left_frame right_frame 30"/> -->

</launch>
